### DISCUSSION TITLE
Why isn't `passHref` enabled by default?

### DISCUSSION THREAD

**Discussion by remy** [vercel/next.js#26729] [Help]
In the codebase I'm looking at  we're seeing a few times when we need to intentionally enable `passHref` on the `Link` component, and we're quickly coming to the decision to *always* use `passHref`.

Which begs the question: why isn't it enabled by default?

I've got to assume there's a good reason - hoping I can pass that back to the team.

**Comment by timneutkens** [comment]
Backwards compatibility. Enabling it by default would break many apps that rely on the current behavior.
